mautrix_signal/portal.py

```diff
@@ -150,6 +156,10 @@ def init_cls(cls, bridge: 'SignalBridge') -> None:
         BasePortal.bridge = bridge
         cls.private_chat_portal_meta = cls.config["bridge.private_chat_portal_meta"]
 
+        for dm in await DisappearingMessage.get_all():
+            if dm.expiration_ts:
+                asyncio.create_task(cls._expire_event(dm))
```

Comment:
This should probably be in a separate method that's run after startup, `init_cls` shouldn't be async